webapp/templates/jury/index.html.twig

```diff
@@ -27,6 +27,7 @@
                     <li><a href="{{ path('jury_judgehosts') }}">Judgehosts</a></li>
                     <li><a href="{{ path('jury_languages') }}">Languages</a></li>
                     <li><a href="{{ path('jury_problems') }}">Problems</a></li>
+                    <li><a href="{{ path('jury_problemset') }}">Problemset</a></li>
```

Comment:
```suggestion
                    <li><a href="{{ path('jury_problems') }}">Problems (Global)</a></li>
                    <li><a href="{{ path('jury_problemset') }}">Problemset</a></li>
```

```suggestion
                    <li><a href="{{ path('jury_problems') }}">Problems</a></li>
                    <li><a href="{{ path('jury_problemset') }}">Current problemset</a></li>
```

```suggestion
                    <li><a href="{{ path('jury_problems') }}">Problems</a></li>
                    <li><a href="{{ path('jury_problemset') }}">Problemset (current contest)</a></li>
```

Without knowing the interface I would not know the difference between the problemset (specific to current contest) and the problems (all contests)

Neither of these suggestions really work, but its the best I could come up with.

Reply:
I pushed another suggestion